src/main/java/util/LocalizationUtil.java

```diff
@@ -16,4 +16,14 @@ public static final Date stringToDateConverter(String string) throws ParseExcept
             return null;
         return dateFormat.parse(string);
     }
+    static SimpleDateFormat simpleDateFormat=new SimpleDateFormat("yyyy-MM-dd hh:mm:ss a");
+    public static final String getFormattedReminderDate(Date date){
```

Comment:
Util should not have methods specific to any other class as it is intended to be used by any class/method.

Reply:
issue resolved in commit 18a0688